### ISSUE TITLE
Genes missing names, IDs, symbols in Rachel's DE objects

### ISSUE THREAD

**Issue by rafecooks** [genialis/bug-reports#30]
Server: BCM
Community: rna-seq
User: Rachel

I selected the collection `p4936_ms2_rna` and went to visualizations.

On DE View, I selected a SE object. Then drew a box around a bunch of genes in the volcano. The resulting popup shows many rows missing gene IDs. 

![screen shot 2017-05-22 at 2 20 12 pm](https://cloud.githubusercontent.com/assets/1234176/26325399/5956d0a6-3efc-11e7-94ac-17765c6004db.jpg)

When the whole lot (28 genes) are added to the Genes widget, and I ask to save a gene set, the number actually present is lower (only 8 in the picture below) than I selected from the Volcano. In other words, the genes missing IDs don't go to the Genes widget and cannot be saved into gene set.

![screen shot 2017-05-22 at 2 20 30 pm](https://cloud.githubusercontent.com/assets/1234176/26325423/6ebe3416-3efc-11e7-98de-92ad3646e5f0.jpg)

[URL](https://bcm.genialis.com/rna-seq/visual_analysis?_b=3905841b-4e06-4684-9098-834606bf6b47&_s=ef6f7c11-f491-47b8-afbc-2408934ea388)

This may or may not be related to an issue with the heatmap. Namely, it seems as if only a small fraction of the genes have any appreciable expression whatsoever (even log2 doesn't help). Is this due to the normalization schema? Is this correct? 

![screen shot 2017-05-22 at 2 26 57 pm](https://cloud.githubusercontent.com/assets/1234176/26325475/a48d734a-3efc-11e7-8978-ca74373c45e9.jpg)

[URL](https://bcm.genialis.com/rna-seq/visual_analysis?_b=3905841b-4e06-4684-9098-834606bf6b47&_s=4bb6da3c-cad9-44ca-8f92-c06e010d8c80)

**Comment by rafecooks** [comment]
@jkokosar @lukaw3d Please see if you can recreate this issue... any thoughts?

**Comment by jkokosar** [comment]
@rafecooks First part of the issue is related to the incompleteness of the Gene KB. Should be resolved soon. The heatmap issue is most probably due to the spike-in normalization used for this dataset. Owner of the data should have some more insights.

**Comment by jkokosar** [comment]
@rafecooks DE objects have been recalculated. There is far less missing genes in the results now, and those that are missing need to be handled at the update to the KB level (e.g. ERCC spike-ins, gene symbols that map to multiple ncbi ids..). Should we close this issue and handle KB issues separately?

**Comment by rafecooks** [comment]
Yeah, we can close the issue and deal with the multiple mappings later. Should we do this as a Github issue or Roadmap item? I prefer to handle it here. In which case, could you write and prioritize the issue how you see fit?

thanks

**Comment by mstajdohar** [comment]
I would keep the issue, but do the multiple mappings later.
